test/css.js

```diff
@@ -154,3 +154,8 @@ QUnit.test( "jQuery.cssNumber", function( assert ) {
 
 	assert.ok( jQuery.cssNumber, "jQuery.cssNumber exists" );
 } );
+
+QUnit.test( "An unsupported jQuery.fn.css(Object,Number) signature", function( assert ) {
+	assert.expect( 1 );
+	assert.ok( ( jQuery( "<div/>" ).css( { left: "100%" }, 300 ) ) );
```

Comment:
Maybe make it clear we test there's no crash?
```suggestion
	assert.ok( ( jQuery( "<div/>" ).css( { left: "100%" }, 300 ), "No crash" ) );
```